Chapter5/frequency.erl

```diff
@@ -0,0 +1,101 @@
+-module(frequency).
```

Comment:
The frequency module is given as an example in the chapter, the changeFreq module is the edited version that includes the features of Exercise 5-2. I can delete the frequency module if needed?